OBAKit/OBAApplication.m

```diff
@@ -131,8 +131,10 @@ - (void)refreshSettings {
     if (!apiURL) {
         [[NSNotificationCenter defaultCenter] postNotificationName:kOBAApplicationSettingsRegionRefreshNotification object:nil];
     }
+    else {
+        self.modelService.obaJsonDataSource = [OBAJsonDataSource JSONDataSourceWithBaseURL:apiURL userID:[OBAUser userIdFromDefaults]];
```

Comment:
good catch. Mind flipping these around? I always find negations harder to read. So this would end up being:

```
if (apiURL) {
    // build obaJSONDataSource
}
else {
    // post notification
}
```